### PR TITLE
port: diffie-hellman

### PR DESCRIPTION
**Pull request by glennj**
[canonical-data.json](https://github.com/exercism/problem-specifications/blob/master/exercises/diffie-hellman/canonical-data.json)

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/bash/blob/master/POLICIES.md)

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -515,6 +515,17 @@
         "parsing",
         "regular_expressions"
       ]
+    },
+    {
+      "slug": "diffie-hellman",
+      "uuid": "14b7ad3a-f3b7-4104-ba4d-3acca25cbb3f",
+      "core": false,
+      "unlocked_by": "two-fer",
```
Comment (kotp) [inline config.json]: Is two-fer the point at which we should unlock this?

Comment (guygastineau) [reply config.json]: Good question.

**Comment by guygastineau** [comment]
I think this is good to go. Are you happy with this this @kotp?

**Comment by kotp** [comment]
Thank you @glennj

**Pull request merged**